test/Test/Common.hs

```diff
@@ -2,15 +2,30 @@
 -- SPDX-License-Identifier: LicenseRef-MIT-TQ
 
 module Test.Common
-  ( withOriginated
+  ( ignore
+  , mkFA2View
+  , checkTokenBalance
+  , withOriginated
+  , originateBaseDao
+  , originateBaseDaoWithConfig
   ) where
 
 import Universum
 
+import qualified Data.Map.Strict as Map
 import Lorentz
+import Lorentz.Test (contractConsumer)
 import Lorentz.Contracts.BaseDAO (baseDaoContract)
 import Lorentz.Contracts.BaseDAO.Types
 import Morley.Nettest
+import Test.Hspec (Expectation)
+import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
+
+import qualified Lorentz.Contracts.BaseDAO as DAO
+
+
+ignore :: Expectation -> Expectation
```

Comment:
Is it the only reason why `hspec` dependency was added? You can change `Expectation` to `Assertion` to avoid adding an additional dependency.

Reply:
I've removed this for now, since we do not use it.